haml/_header.html.haml

```diff
@@ -3,6 +3,12 @@
   sign_in_out ||= {}
   main_content_anchor_link ||= "#cads-main-content"
   homepage_url ||= "/"
+  button_attr = {
+    class: %w[cads-header__search-reveal js-cads-search-reveal cads-icon_search],
+    title: t("cads.header.open_search"),
```

Comment:
is this value needed now? Given your JS switches it below?

Reply:
It is needed - for the initial render